Hunting Queries/SlackAudit/SlackAuditApplicationsInstalled.yaml

```diff
@@ -0,0 +1,10 @@
+id: 9a2778a1-ac23-40f2-9579-a0c344127c89
+name: SlackAudit - Applications installed 
+description: |
+  'This query detects application installation fact.'
+tactics: Execution
+query: |
+  SlackAudit
+  | where TimeGenerated > ago(lbtime)
+  | where DvcAction =~ 'app_installed'
+  | summarize by SrcUserName, EntityAppName
```

Comment:
Hunting queries should also have the entityMappings - https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide